src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature

```diff
@@ -10,10 +10,10 @@ Feature: #Enter feature name here
     Then a drop down list should appear in the "Person" row
     Then the list in the "Person" row should contain "Local Persons, ULAN Persons"
       And clicks on the Create button
-    Then the vocabulary name in the titlebar should contain "Local Person"
+    Then the titlebar should contain "Local Person"
```

Comment:
I recall that if you were operating on a feature/topic branch (always a good idea :), simply changing that branch and pushing again would also update the PR. By analogy, it's conceivable that simply making one more local commit, to revert the two changed lines in that feature file, and pushing to cesarvh:master might do the trick? (Am thinking of what might be simplest here ...)